source/creating_packages.rst

```diff
@@ -0,0 +1,368 @@
+
+Create the directory ``~/.conda-smithy`` (in your home directory). 
+
+.. code-block:: bash
+
+    $ cd ~
+    $ mkdir .conda-smithy
+
+Copy tokens to ``anaconda.token``, ``azure.token`` and ``github.token`` 
+to ``~/.conda-smithy``:
+
+.. code-block:: bash
+
+    $ cp /keybase/team/dama/anaconda.token .conda-smithy/
+    $ cp /keybase/team/dama/azure.token .conda-smithy/
+    $ cp /keybase/team/dama/github.token .conda-smithy/
+
+Create Directory for Packages
+-----------------------------
+
+Naming and location of the working directory is aribtrary. We assume
+that the directory `packages` in `HOME` directory is used:
+
+.. code-block:: bash
+
+    $ mkdir ~/packages
+    $ cd ~/packages
+
+Clone *event-model-feedstock* repository from GitHub
+----------------------------------------------------
+
+The *event-model-feedstock* needs to be cloned only once. In the process of
+creating packages we will be using file ``conda_build_config.yaml`` located
+in `~/packages/event-model-feedstock/recipe/` directory. If you already
+have a clone of the feedstock, pull the latest version of the
+*event-model-feedstock* since GitHub repository may be updated.
+
+.. code-block::
+
+    $ cd ~/packages
+    $ git clone https://github.com/nsls-ii-forge/event-model-feedstock.git
+
+
+Generate and Edit meta.yaml Recipe Configuration File
+=====================================================
+
+Create directory for your package `(your-package-name)`:
+
+.. code-block:: bash
+
+    $ cd ~/packages
+    $ mkdir (your-package-name)
+    $ cd (your-package-name)
+
+(You are now in the directory ``~/packages/(your-package-name)``)
+
+.. note::
+
+    It is important that each command is started with the correct
+    home directory. Since it is easy to lose track of
+    directory changes, all the paths will be specified 
+    relative to HOME directory.
+
+
+There are several ways to create the package recipe:
+
+- generate recipe from existing PyPI package;
+
+- manually create ``meta.yaml`` recipe file using ``conda skeleton``;
+
+- use existing ``meta.yaml`` recipe file.
+
+Generate recipe from PyPI package
+---------------------------------
+
+If the package ``(your-package-name)`` is available from PyPI, 
+generate the recipe from the existing package:
+
+.. code-block:: bash
+
+    $ cd ~/packages/(your-package-name)
+    $ conda skeleton pypi (your-package-name) --noarch
+
+Check if ``meta.yaml`` was successfully created in
+``~/packages/(your-package-name)/(your-package-name)`` directory.
+
+Edit ``meta.yaml`` file:
+
+Remove all entries from host section except python and pip.
+Add ``requires`` and ``commands`` to the ``test`` section.
+
+.. code-block::
+
+  requires:
+    - pytest
+  commands:
+    - pytest --pyargs sixtools.tests
+
+Remove the following lines from the ``about`` section:
+
+.. code-block::
+
+  doc_url: ""
+  dev_url: ""
+
+Additional steps:
+
+  -- Add license file name ``LICENSE`` to ``about`` section.
+  -- Update URL.
```

Comment:
-- Update ``home`` URL in ``about`` section to point to the package's GitHub repository.